### ISSUE TITLE
[lux-styleguidist branch] Failed to resolve component text-style

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/approvals#995]
### Expected behavior
On the lux-styleguidist branch, I can see the styles from the lux `lux-text-style` component.

### Actual behavior
All the text looks the same.  I get the following warning in the console in firefox:

```
[Vue warn]: Failed to resolve component: text-style
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <App>
```

### Steps to replicate

1. Checkout the lux-styleguidist branch
2. Follow [these yalc instructions](https://github.com/pulibrary/lux-styleguidist?tab=readme-ov-file#using-a-local-copy-of-lux-styleguidist-in-your-project)

### Implementation notes, if any

* The component should now be called `lux-text-style`.  It's also possible that we need to make changes in the lux-styleguidist repo and this repo to properly set up lux in this project.

**Comment by sandbergja** [comment]
A few things I've noticed so far:

- [ ] We are not using the rollup-plugin-vue dependency in lux-styleguidist, we can remove it
- [ ] We are not using the src/entry.js in lux-styleguidist, we can remove it
- [ ] When we include LuxGallery in the bundle, the bundle size gets large and the first line of dist/lux-styleguidist.mjs includes this (which throws an error on `import Mr`, since Vue 3 doesn't have a default export, that was a Vue 2 thing):
```
import Mr, { openBlock as q, createElementBlock as mt, createElementVNode as St, toDisplayString as Ht, createCommentVNode as _t, renderSlot as fe, createBlock as jt, resolveDynamicComponent as de, withCtx as Nt, normalizeClass as Vt, Fragment as Lr, renderList as jr, resolveComponent as Lt, createVNode as ce, normalizeStyle as Fr, watch as lr, effectScope as $r, reactive as Gr, computed as Ur, createTextVNode as Yn } from "vue";
```
- [ ] We should externalize the vue library (partially revert my last PR)
- [ ] sizeMixin is not a component, and therefore does not have a name.  We shouldn't treat it like a component in index.js
- [ ] In approvals, rather than creating multiple identical Vue apps, it seems like the console warnings want a factory function that returns them like so:
```
// Create a factory function that will create vue
// apps, which we can then mount to any element with
// the class .lux
const app = createApp({});
lux.install();
app.use(lux);
app.component('event-date-modal', eventDateModal)
    .component('event-title-input-wrapper', eventTitleInputWrapper)
    .component('hours-calculator', hoursCalculator)
    .component('travel-estimate-form', travelEstimateForm)
    .component('travel-request-button', travelRequestButton)
    .component('travel-request-date-pickers', travelRequestDatePickers)
const createMyApp = () => createApp(app)
```

**Comment by sandbergja** [comment]
closed by #996
